docs/ecosystem/roadmap.md

```diff
@@ -0,0 +1,38 @@
+---
+id: roadmap
+title: Roadmap
+---
+
+## ORY Roadmap
+
+If you would like a quick overview of whats coming next to ORY, you can do so at our github project page.  
+You can see what we are working on right now, what issues are prioritized and what progress has been made in the past.  
+[Follow this link for the complete Project Roadmap](https://github.com/orgs/ory/projects/8).
```

Comment:
I think we can remove this as it's linked above.

Reply:
done